string TEST_MICROPHONE = test-microphone
string RELAX_RIGHT_ARM = relax-right-arm
string RELAX_LEFT_ARM = relax-left-arm
string OPEN_RIGHT_HAND = open-right-hand
string OPEN_LEFT_HAND = open-left-hand
string CLOSE_RIGHT_HAND = close-right-hand
string CLOSE_LEFT_HAND = close-left-hand
string STOP_EXECUTION = stop-execution
string UNDO = undo-last-command
string DELETE_ALL_STEPS = delete-all-steps
string DELETE_LAST_STEP = delete-last-step
string FREEZE_RIGHT_ARM = freeze-right-arm
string FREEZE_LEFT_ARM = freeze-left-arm
string RECORD_OBJECT_POSE = record-object-pose
string CREATE_NEW_ACTION = create-new-action
string EXECUTE_ACTION = execute-action
string NEXT_ACTION = next-action
string PREV_ACTION = previous-action
string SAVE_ACTION = save-action
string EDIT_ACTION = edit-action
string SAVE_POSE = save-pose
string START_RECORDING_MOTION = start-recording-motion
string STOP_RECORDING_MOTION = stop-recording-motion
string EXECUTE_GENERATED_ACTION = execute-generated-action
string CALCULATE_POSE_DISTRIBUTION = calculate-pose-distribution
string CONTINUE_EXECUTION = continue-execution
string UNRECOGNIZED = unrecognized
string command
